packages/cpp-debug/src/browser/cpp-memory-widget.tsx

```diff
@@ -84,24 +85,46 @@ export class MemoryView extends ReactWidget {
         </div>;
     }
 
-    protected renderSearchField(): React.ReactNode {
-        return <div className='t-mv-search-container'>
-            <div className='label t-mv-search-label'>Memory Address</div>
-            <input id='t-mv-search' type='text' onKeyUp={this.doRefresh} />
-        </div>;
+    protected renderInputContainer(): React.ReactNode {
+        return <div id='t-mv-wrapper'>
+            <div className='t-mv-group'>
+                <span className='t-mv-input-group'>
+                    <label className='t-mv-label'>Memory Address</label>
```

Comment:
Since there is already a "Memory Address" placeholder in the input box, maybe this could be removed to save some space?

Reply:
Sure I think it's either or, either we display a label, or we have the placeholder.
Since we want to save space, I'll only include the placeholder as you mentioned.